### ISSUE TITLE
Suggestion: Add keybind for escort formation

### ISSUE THREAD

**Issue by Wuzzy2** [naev/naev#1073]
I suggest to add a button to select an escort formation, not just a button you must click and that you can easily overlook.

Literally every other action in the game has a keypress, so it doesn't make sense to make an exception here.

Version: 0.8.0 beta

**Comment by onpon4** [comment]
There's no keybind because escord formation was only recently added, and it's also exclusively a Lua thing that has no interaction with the C code (Lua code can't define key bindings). Actually there was discussion of effectively deprecating the rest of the escort keys because they're a steep learning curve and not ideal control.

**Comment by Wuzzy2** [comment]
Noooo, please at least keep the escort keys that we have now, because forcing the player to use the mouse (with no alternative whatsoever) is worse.
Being able to order your escort with a single keypress is a GOOD thing, not a bad thing. But yes, you have to learn it first, obviously, but that's a different topic. There's no tutorial for that, however. How about adding a small (optional!) tutorial mission after you buy the first fighter bay? But that's off-topic ...

As for on-screen buttons, you can always add them on top of the existing system, no reason that both exclude each other.

**Comment by onpon4** [comment]
That's rather subjective. I myself find it annoying to have to look away from my screen to find the key I need to press to make my ships attack. In that sense I would find a button on the screen to be a lot more convenient.

Also, there's a limit to how many keys are on the keyboard. We're already at a point where we're probably overusing the keyboard.

> There's no tutorial for that, however. How about adding a small (optional!) tutorial mission after you buy the first fighter bay?

I don't think relying on tutorials is a good way to design things. Much better is to do things that naturally lead to teaching the player how to play. it's a lot harder to do that when playing involves pressing a bunch of keys on the keyboard.

Anyway, we're not about to just suddenly remove existing keys, so don't worry about that. But if / when we do, it won't be to require a mouse but to *consolidate* the existing four keys into a single key that brings up a menu. Probably a menu that would handle fleet formations too if I had to guess.

**Comment by Wuzzy2** [comment]
> Probably a menu that would handle fleet formations too if I had to guess.

Good idea! But even then I would keep the one-keypress shortcuts. Keep them unset by default if this worries you so much. There aren't that many escort commands anyway.

**Comment by onpon4** [comment]
See #864 for the relevant issue.
